docket/api.py

```diff
@@ -1,20 +1,28 @@
 from flask import Flask, Blueprint, Response
-from flask_restful import Resource, Api
+from flask_restful import Api
 
-from resources.pcap import PcapApi, PcapUri, RawQuery
-from resources.stats import Stats
+from resources.query import QueryRequest, ApiRequest, RawRequest
+
+from config import Config
 
 # Declare the blueprint
 api_bp = Blueprint('api', __name__)
-api = Api(api_bp)
+api    = Api(api_bp)
```

Comment:
Let's not put extra spaces to align assignments. See [this stackoverflow](https://stackoverflow.com/questions/101958/code-formatting-is-lining-up-similar-lines-ok) for related discussion.

Reply:
Also, this violates [PEP8](https://www.python.org/dev/peps/pep-0008/#pet-peeves)

Reply:
Fair enough.